team-roles/readme.md

```diff
@@ -0,0 +1,15 @@
+# Team roles
+
+## Scrum roles
+
+TODO: link to role descriptions
+
+The Scrum Master can also be accountable for PR turnaround, assigning reviewers and following up with them to make sure that reviews are done in a timely fashion.
+
+## Other roles
+
+On top of the roles defined in the Scrum process, defining one or some of the virtual roles below can
+help the team navigate specific situations successfully:
+
```

Comment:
Have these roles been discussed with the Agile Engineering Fundamentals Champions? I would want to make sure they are in agreement as I have personally not worked on a project with these roles. 
We usually absorb responsibility for guidelines to everyone on the team and ultimate responsability to the Tech Lead and co-lead on the customer side. For the "Merge master" we leave it up to the person who owns the PR to complete their PRs once they have addressed all comments etc, good CI/CD and testing usually helps us manage most of this.

Reply:
@fnocera I will remove this file for the time being and discuss with the champions first; the suggestion to consider these additional roles comes from real life experiences and successful practices so I'd argue that the approach you describe above may vary. Thanks for the feedback.